lib/dal/src/workspace_snapshot/edge_weight.rs

```diff
@@ -23,6 +23,11 @@ pub enum EdgeWeightKind {
     Action,
     /// A function used by a [`SchemaVariant`] to perform an action that affects its resource
     ActionPrototype,
+    /// Used to indicate that an [`ActionPrototype`] should always run
+    /// after a successful action run
+    ///
+    /// For example, we currently always run a refresh action after a successful create
+    ActionRunsAfter,
```

Comment:
Gonna nit-pick the name here. I think the edge `A -> B` would be `When A finishes, it triggers B to run`. But the name `ActionRunsAfter` implies a relationship that is the opposite direction from the edge I described `A -- ActionRunsAfter --> B` would be `When B finishes, it triggers A to run`. I think the `ActionRunsAfter` name also brings up the question of whether the feature is "you can specify that this action always runs after any of A, B, or 
C succeed", or if it is "you can specify that whenever this action succeeds, it will cause the following action(s) to run". (Either way can be implemented on top of edges in either direction, but it's probably best to start out with the user facing feature, and the underlying data to match in their directionality.)